src/main/java/org/jenkinsci/plugins/displayurlapi/actions/AbstractDisplayAction.java

```diff
@@ -61,20 +61,27 @@ DisplayURLProvider lookupProvider(StaplerRequest req) {
     }
 
     DisplayURLProvider lookupProvider() {
```

Comment:
TODO: it might make sense to expose this as a method in `DisplayURLProvider` for use in tests. Perhaps we can reuse `getPreferredProvider`, but I would need to double-check whether it is still used by anything.

Reply:
Done in [a123356](https://github.com/jenkinsci/display-url-api-plugin/pull/202/commits/a12335684612c29376ff1b1dc82bc475f702c30d).